gold/options.h

```diff
@@ -1217,6 +1217,11 @@ class General_options
 	      N_("(nanoMIPS only) Relax R_NANOMIPS_LO12 relocation"),
 	      N_("(nanoMIPS only) Do not relax R_NANOMIPS_LO12 relocation"));
 
+  DEFINE_bool(expand_long_balc_bc, options::TWO_DASHES, '\0', false,
+              N_("(nanoMIPS only) Expand balc/bc if offset is >= 0xff0000 for "
```

Comment:
Strictly speaking, the description is not accurate. I'd opt for a more descriptive (if imprecise) wording like: Expand long branches targeting the furthest 64K at either ends of the permissible26-bit signed range.

The other option is to just put the H/W bug number once we have it and leave the user to refer to other sources. For references, see the description of various --fix* options for ARM cores.